cmd/traffic_layout/engine.cc

```diff
@@ -119,6 +120,7 @@ help_message(const int versionflag, const int runflag, const int cleanflag, cons
   std::cout << "Remove:   --remove(-r)   (remove the ts_runroot sandbox)\n" << std::endl;
   std::cout << "==flag=======key=========description======================================" << std::endl;
   std::cout << "force:    --force   (force to create ts_runroot, only works with init)\n" << std::endl;
+  std::cout << "relocate: --relocate   (relocate the directory creating a good form during copy)\n" << std::endl;
```

Comment:
You are basically printing a string literal here. You can replace all this with a [raw string literal](http://en.cppreference.com/w/cpp/language/string_literal).